python/dgl/model_zoo/chem/README.md

```diff
@@ -0,0 +1,39 @@
+# DGL for Drug Discovery
+
+With atoms being nodes and bonds being edges, molecular graphs are among the core objects for study in drug discovery. 
+As drug discovery is known to be costly and time consuming, deep learning on graphs can be potentially beneficial for 
+improving the efficiency of drug discovery [1], [2].
+
+With pre-trained models and training scripts, we hope this dgl model zoo on drug discovery will be helpful for both
+the chemistry community and the deep learning community to further their research.
+
```

Comment:
Add additional dependencies needed for using chem related functions, including the package names, versions, the recommended ways for installation and the links to the official website.

Reply:
This should include `RDKit`, `pandas`, `requests`.